GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ExamplesPanelViewModel.cs

```diff
@@ -48,34 +52,34 @@ public GalaxyExample SelectedExample
 
         public ExamplesPanelViewModel()
         {
-            ExampleGalaxies.Add(new GalaxyExample(GalaxyType.Smooth));
-            ExampleGalaxies.Add(new GalaxyExample(GalaxyType.Features));
-            ExampleGalaxies.Add(new GalaxyExample(GalaxyType.Star));
-
             LoadCommands();
         }
 
         public void LoadCommands()
         {
             OpenPanel = new CustomCommand(SlidePanel, CanOpen);
             TogglePanel = new CustomCommand(SlidePanel, CanToggle);
-            SelectionChanged = new CustomCommand(SelectExample, CanChoose);
+            SelectItem = new CustomCommand(OnToggleItem, CanSelectItem);
+            UnselectItem = new CustomCommand(OnToggleItem);
         }
 
-        public void SelectExample(object sender)
+        public void OnToggleItem(object sender)
         {
-            if (sender == null)
```

Comment:
Is there a reason this is no longer needed?

Reply:
Yes, this was originally used when a `ListView` element was used in place of all galaxy examples. Looks like I need to remove the `ICommand` definition above as well. 

`SelectItem` and `UnselectItem` now handle this logic as we are no longer using the finicky `ListView`.